### PR TITLE
Refactor javascript files and the popup

### PR DESCRIPTION
**Pull request by thePeras**
Decided to refactor some little things 🤕 

Now the popup is this way. Hope you agree to the changes!

Screenshot 2024-11-02 at 17 04 12

What do you think to change the texts to?
- Barra de navegação
- Esconder atalhos
- Nova fonte de letra
- Relembrar sessão

### CODE REVIEW

File: content-scripts/modules/icons/index.ts
Comment (toni-santos) [inline content-scripts/modules/icons/index.ts:1]: there are some parts of this file that could be reimplemented in jsx

File: content-scripts/modules/options/addStyle.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    // If the element already exists, remove it
+    removeElement(id);
+
+    const head = document.querySelector("head");
+    if (head) {
+        const style = document.createElement("style");
+        style.id = id;
+        style.textContent = css;
+        head.appendChild(style);
+    }
```
Comment (toni-santos) [inline content-scripts/modules/options/addStyle.ts:14]: this could be tsx

**Review by toni-santos — CHANGES_REQUESTED** [review]
this is very good 🤩, we could start migrating some of these things to tsx. also what do you think about separating options further by page? it could be helpful as we increase customisation.

**Comment by thePeras** [comment]
> What do you think about separating options further by page? it could be helpful as we increase customisation.

I agree. I put a comment in the file, so when the options increase next time, could be an improvement

**Review by toni-santos — APPROVED** [review]

**Pull request merged**